docs/question.md

```diff
@@ -272,12 +272,6 @@ To disable partial credit for a question, set `"partialCredit": false` in the `i
 
 In general, it is strongly recommended to leave partial credit enabled for all questions.
 
-## Preventing questions from locking when full credit is achieved
-
-Currently, PrairieLearn will lock a question and prevent students from submitting revised answers as soon as they score 100% on the problem. This may have negative side effects for questions where students would like to continue to refine their answer (for example, by adding additional comments to their code for staff reviewers to see). A workaround for this is to enable partial credit and only award at most 99% in your grader configuration for the question. You can explain to students that the last 1% of the grade will come from staff reviews for integrity.
-
-In the future, PrairieLearn may add an option to prevent this lock from occurring even with a 100% grade. Refer to this issue: https://github.com/PrairieLearn/PrairieLearn/issues/3191
-
```

Comment:
I removed this comment since it is no longer true, but if you believe it's better to replace it with a comment on setting manual points I'm happy to do it.